libafl_concolic/test/smoke_test.sh

```diff
@@ -0,0 +1,20 @@
+set -eux;
+
+# this test intends to ...
+# 1. compile symcc with the rust/tracing backend
+# 2. compile a program using this symcc
+# 3. run the program, capturing constraints
+# 4. print the constraints in human readable form for verification
+
+rm -rf symcc_build
+mkdir symcc_build
+
+cd symcc_build
+cmake -G Ninja -DRUST_BACKEND=YES ../../libafl_symcc 
+ninja
+cd ..
+symcc_build/symcc ../libafl_symcc/test/if.c -o "if"
+
+cargo run -p dump_constraints -- --plain-text --output constraints.txt -- ./if < if_test_input
+
+cat constraints.txt
```

Comment:
Want to add a github actions thingy for that?

Reply:
yes this should be the ultimate goal, but I don't want to do it until the structure of where which component resides in the end is final, because I expected there will be quite a bit of changes.